tests/e2e/specs/BoundaryEscalationEvent.spec.js

```diff
@@ -0,0 +1,103 @@
+import {
+  dragFromSourceToDest,
+  getComponentsEmbeddedInShape,
+  getElementAtPosition,
+  removeIndentationAndLinebreaks,
+  waitToRenderAllShapes,
+} from '../support/utils';
+import { nodeTypes } from '../support/constants';
+
+const boundaryEventSelector = '.main-paper ' +
+    '[data-type="processmaker.components.nodes.task.Shape"] + ' +
+    '[data-type="processmaker.components.nodes.boundaryEvent.Shape"]';
+
+describe.only('Boundary Escalation Event', () => {
+  beforeEach(() => {
+    cy.loadModeler();
+  });
+
+  it('Render a boundary escalation event', function() {
+    if (Cypress.env('inProcessmaker')) {
+      this.skip();
+    }
+
+    const taskPosition = { x: 200, y: 200 };
+    dragFromSourceToDest(nodeTypes.task, taskPosition);
+
+    const boundaryEscalationEventPosition = { x: 260, y: 260 };
+    dragFromSourceToDest(nodeTypes.boundaryEscalationEvent, boundaryEscalationEventPosition);
+
+    getElementAtPosition(boundaryEscalationEventPosition).click();
+
+    const boundaryEscalationEventXML = '<bpmn:boundaryEvent id="node_3" name="New Boundary Escalation Event" attachedToRef="node_2"><bpmn:escalationEventDefinition /></bpmn:boundaryEvent>';
+    
+
+    cy.get('[data-test=downloadXMLBtn]').click();
+
+    cy.window()
+      .its('xml')
+      .then(removeIndentationAndLinebreaks)
+      .then(xml => {
+        expect(xml).to.contain(boundaryEscalationEventXML);
+      });
+  });
+
+  it('can successfully undo/redo after dragging onto invalid (empty) space ', function() {
```

Comment:
You can probably leave all these tests, but as per the issue, it mentions:

> Automated test:
> Drag event type onto valid drop target.
> Download XML.
> Assert that the XML contains the snippet required from the BPMN docs.

So I think the only test we really need is "Render a boundary escalation event", especially since @JCook21 has a PR up for cleaning up these related tests.